core/src/test/java/org/infinispan/distribution/DistAsyncFuncTest.java

```diff
@@ -65,7 +66,8 @@ protected void createCacheManagers() throws Throwable {
             protected <T> CompletionStage<T> performRequest(Collection<Address> targets, ReplicableCommand command,
                                                             ResponseCollector<T> collector,
                                                             Function<ResponseCollector<T>,
-                                                               CompletionStage<T>> invoker) {
+                                                                       CompletionStage<T>> invoker,
```

Comment:
indentation?

Reply:
That's IntelliJ aligns it for me if it's on a separate line, but it fits on the `Function<` line so I moved it